doc/usage/extensions/math.rst

```diff
@@ -15,7 +15,8 @@ Math support for HTML outputs in Sphinx
    So mathbase extension is no longer needed.
 
 Since mathematical notation isn't natively supported by HTML in any way, Sphinx
-gives a math support to HTML document with several extensions.
+gives a math support to HTML document with several extensions.  These use the
+Sphinx :rst:dir:`directives <math>` and :rst:role:`roles <math>`.
```

Comment:
And I think both `directive` and `role` should be singular form.

Reply:
Good point, fixed (and also added "math" so that it reads "math directive and role")